extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php

```diff
@@ -157,13 +157,19 @@ public function __construct( PathBuilder $pathBuilder ) {
 
 	public function configureRobotsBuilder( RobotsTxt $robots ) {
 		global $wgEnableSitemapXmlExt,
+		       $wgFandomBaseDomain,
 		       $wgRobotsTxtBlockedWiki,
 		       $wgSitemapXmlExposeInRobots,
 		       $wgServer,
 		       $wgScriptPath,
 		       $wgRequest;
 
-		if ( !$this->accessAllowed || !empty( $wgRobotsTxtBlockedWiki ) ) {
+		if (
+			!$this->accessAllowed ||
```

Comment:
Wouldn't it be better to incorporate the domain check into the `accessAllowed` flag? This would allow us to still use the `forcerobots` query param:
```
$this->accessAllowed = ( ( $wgWikiaEnvironment === WIKIA_ENV_PROD && wfGetBaseDomainForHost( $wgServer ) === $wgWikiaBaseDomain ) || $wgRequest->getBool( 'forcerobots' ) );
```

Reply:
It'll also add `noindex,nofollow` meta and that's good.

Reply:
https://github.com/Wikia/app/pull/15932/commits/5aff7d7394b4b60614e324ac702e262e0fbaa198